app/api/helpers/utilities.py

```diff
@@ -96,7 +96,7 @@ def get_filename_from_cd(cd):
 
 def write_file(file, data):
     """simple write to file"""
-    fp = open(file, 'w')
+    fp = open(file, 'wb')
```

Comment:
Instead of changing the write mode, what if we mantain the invariant of writing a string? If the original data is a string, there is no problem, but if the data consists of bytes, can't we do something like `fp.write(data.decode())`?

Reply:
:+1: